samples/snippets/snippets_test.py

```diff
@@ -52,21 +52,21 @@ def iam_user():
 
 @pytest.fixture()
 def secret(client, project_id):
-    parent = client.project_path(project_id)
+    parent = f"projects/{project_id}"
```

Comment:
Could you run `black` over this directory so the single/double quote usage is consistent?

Reply:
done